standards/application/telemetry.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
I think we need `metrics` section. There can be 2 types of them: general and implementation specific.

As a basic I think following should be specified: 
- timestamps of receiving and sending message;
- size of message packets;
- content topics / pubsub topics of the messages;
- code logs of errors and warnings;
- peer ID or sender and receiver;
- message hashes;

In case we collect multi addresses we can derive IP and try to see correlation between geographical presence and availability. This approach is questionable and unless we need it we should drop it.

Reply:
Relay metrics needed:
- mesh state: peers discovered, connections established;
- any errors with peers;

This is a follow-up from today's Bug Bash meeting. 

What else can we add, @jm-clius @chaitanyaprem ?